ingestion/src/metadata/utils/powerbi_client.py

```diff
@@ -0,0 +1,130 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+REST Auth & Client for PowerBi
+"""
+import traceback
+from typing import Tuple
+
+import msal
+
+from metadata.ingestion.ometa.client import REST, ClientConfig
+from metadata.utils.logger import ingestion_logger
+
+logger = ingestion_logger()
```

Comment:
we should use `utils_logger` here instead of `ingestion_logger`

Reply:
ok